src/Browser.php

```diff
@@ -770,7 +771,7 @@ private function withOptions(array $options)
      * @param string                         $url
      * @param array                          $headers
      * @param string|ReadableStreamInterface $body
-     * @return PromiseInterface<ResponseInterface,\Exception>
+     * @return PromiseInterface<ResponseInterface,\Exception,ConnectionInterface>
```

Comment:
What is this supposed to do? The `PromiseInterface` does not currently nor the foreseeable future have a third template type.